src/features/environment.js

```diff
@@ -78,7 +78,6 @@ export const EnvironmentMixin = (ModelViewerElement) => {
     [$tick](time, delta) {
       super[$tick](time, delta);
       const camera = this[$scene].getCamera();
-      this[$scene].skysphere.position.copy(camera.position);
```

Comment:
Why is this no longer needed? Wouldn't this change the skysphere behavior to 'zoom' when zooming in on the model, as opposed to it's current implementation? I think there are also some scenarios where the skysphere may not be big enough, but I think that was due to panning in orbit controls

Reply:
Speaking subjectively, the behavior without this line looks more natural to me when moving towards / away from the model.

Reply:
I agree, and that's what we had originally -- I believe @mrdoob mentioned that a skybox moving with the camera was the common/expected skybox behavior though

Reply:
I asked @smalls for an additional take and he prefers skybox not zooming, so I will revert this line

Reply:
Backreferencing #100 so that we can track this discussion as part of that issue